experiments/kompanion/cmd/export/export.go

```diff
@@ -90,10 +95,110 @@ func init() {
 
 	ExportCmd.Flags().StringArrayVarP(&targetObjects, targetObjectsFlag, "", []string{}, "object name prefix to target. Targets all if empty. Can be specified multiple times.")
 	ExportCmd.Flags().StringArrayVarP(&ignoreObjects, ignoreObjectsFlag, "", []string{}, "object name prefix to ignore. Excludes nothing if empty. Can be specified multiple times.")
+
+	ExportCmd.Flags().IntVarP(&workerRountines, workerRoutinesFlag, "", 10, "Configure the number of worker rountines to export namespaces with. Defaults to 10. ")
+}
+
+var (
+	reportName string
```

Comment:
Nit: Global variables are generally an anti-pattern, better to use a struct to hold them.